src/main/groovy/com/ullink/NuGetSpec.groovy

```diff
@@ -75,7 +75,6 @@ class NuGetSpec extends Exec {
         }
 
         setDefaultMetadata ('id', project.name)
-        setDefaultMetadata ('title', project.name)
```

Comment:
Yes, it's not needed.
https://docs.nuget.org/create/nuspec-reference
"If none is specified, the ID is used instead."